opcodes/src/lib.rs

```diff
@@ -58,6 +60,92 @@ pub enum Opcode {
     MULSSECP256K1,
 }
 
```

Comment:
It seems like the purpose of this is to limit the values of the opcodes to 256-bit values. This avoids a need to change the compiler in order to do so. I'll take it, but it would be nice to update this to just be the identity function, although this would require changing the compiler.